cmake/compiler/xcc-clang/compiler_flags.cmake

```diff
@@ -1,11 +1,18 @@
 # SPDX-License-Identifier: Apache-2.0
 
 include(${ZEPHYR_BASE}/cmake/compiler/clang/compiler_flags.cmake)
+include(${ZEPHYR_BASE}/cmake/compiler/xcc/target.cmake)
```

Comment:
Wow, that's a pretty big change... so this file was basically not tested yet?

Reply:
No longer needed, so removed this line.